sdcm/tester.py

```diff
@@ -1526,7 +1526,9 @@ def run_fullscan(self, ks_cf, db_node, page_size=100000):
 
         cmd_select_all = 'select * from {}'
         cmd_bypass_cache = 'select * from {} bypass cache'
-        cmd = random.choice([cmd_select_all, cmd_bypass_cache]).format(ks_cf)
+        cql_timeout_minutes = str(random.choice([2, 4, 8, 30, 120, 300]))
```

Comment:
why not `random.choice(range(2, 300))` ?

Reply:
because it is less interesting to check an average timeout duration of 150 minutes. the full scans can last either few minutes or much longer (depend on selected table) so the idea is taking these specific time points to challenge both successful and unsuccessful query to all relevant tables and duration in a balanced probability.